src/app/settings/user/template.html

```diff
@@ -98,6 +98,26 @@
         <km-spinner-with-confirmation [isSaved]="isEqual(settings.selectedProjectID, apiSettings.selectedProjectID)"
                                       fxFlexAlign=" center"></km-spinner-with-confirmation>
       </div>
+      <div fxLayout="row">
+        <div class="entry-label"
+             fxLayoutAlign=" center">Project Landing Page
+        </div>
+        <mat-button-toggle-group [(value)]="selectedProjectLandingPage"
```

Comment:
Buttons size looks inconsistent IMO wdyt?
It should be of same width as above -> System, Dark?

@cschieder ?

<img width="737" alt="buttons-size" src="https://user-images.githubusercontent.com/17727069/177502709-294f0a35-c660-4e08-940e-efa708046ab7.png">

Reply:
I don't think we should make them the same width, it looks strange
![image](https://user-images.githubusercontent.com/49145827/177564616-f59ae2a1-239d-4377-8462-ae4f8018e0d3.png)


- Let's use the same padding etc is so the width is dynamic based on content like in the following screenshot
- on your screenshot I believe the project and cluster icon look too large/a bit larger then the rest? That's odd, please take another look
- also maybe we should change the system icon to something different like a cogwheel - wdyt?

![image](https://user-images.githubusercontent.com/49145827/177565000-c192675b-9f42-4d65-b68f-3189dc47103f.png)



By the way I'd remove the "Account" headline and the thin line separator so it looks like this,
![image](https://user-images.githubusercontent.com/49145827/177564923-da5b72cc-e58b-4514-a22f-8158edab81a5.png)

Reply:
Agreed on:
- `CogWheel` icon on System
- Yes! adding button on `Clusters` looks good to me